applications/nrf_desktop/src/util/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+target_sources_ifdef(CONFIG_DESKTOP_HID_EVENTQ
+		     app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/hid_eventq.c)
```

Comment:
cmake indent is 2 space and weirdly I feel like I just commented on this exact thing in another PR

Reply:
Yes, the another PR needs to go in first (that's why I now added DNM here)